src/System.Diagnostics.DiagnosticSource/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      net45-Windows_NT;
```

Comment:
Correct, I cannot build it on net45. Do you suggest to remove build configuration for net45 until it's enabled?